### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

File: tap-snapshots/test/federation.js.test.cjs [hunk 1]
```diff
@@ -1,46 +0,0 @@
-/* IMPORTANT
```
Comment (jonnydgreen) [inline tap-snapshots/test/federation.js.test.cjs:1]: Was this one a redundant snapshot? :)

File: tap-snapshots/test/gateway/remote-services.js.test.cjs [hunk 2]
```diff
@@ -1,14 +0,0 @@
-/* IMPORTANT
```
Comment (jonnydgreen) [inline tap-snapshots/test/gateway/remote-services.js.test.cjs:1]: Was this one a redundant snapshot? :)

File: test/subscription.js [hunk 3]
```diff
@@ -1200,7 +1201,7 @@ test('subscription server sends correct error if there\'s a graphql error', t =>
       payload: {
         query: `
           subscription {
-            notificationAdded {
+            notificationAdded {,
```
Comment (jonnydgreen) [inline test/subscription.js:1204]: Is this change intentional?

File: test/subscription.js [hunk 4]
```diff
@@ -1245,7 +1246,7 @@ test('subscription server sends correct error if there\'s a graphql error', t =>
   })
 })
 
-test('subscription server exposes pubsub', t => {
+test('subscription server exposes pubsub', { only: true }, t => {
```
Comment (jonnydgreen) [inline test/subscription.js:1249]: Should we remove the `only: true` here? :)

File: package.json [hunk 5]
```diff
@@ -31,44 +31,44 @@
...
     "fastify-plugin": "^5.0.0",
-    "graphql-jit": "0.8.6",
+    "graphql-jit": "0.8.7",
     "mqemitter": "^6.0.0",
     "p-map": "^4.0.0",
     "quick-lru": "^7.0.0",
     "readable-stream": "^4.0.0",
     "safe-stable-stringify": "^2.3.0",
-    "secure-json-parse": "^2.7.0",
+    "secure-json-parse": "^3.0.0",
```
Comment (jonnydgreen) [inline package.json:69]: I'm guessing this one means it's a semver major change? :)

**Review by jonnydgreen — COMMENTED** [review]
Functionally LGTM - just had a few questions! :)

File: tap-snapshots/test/federation.js.test.cjs (hunk 1 above)
Comment (mcollina) [reply tap-snapshots/test/federation.js.test.cjs:1]: yes

File: tap-snapshots/test/gateway/remote-services.js.test.cjs (hunk 2 above)
Comment (mcollina) [reply tap-snapshots/test/gateway/remote-services.js.test.cjs:1]: yes

**Review by jonnydgreen — APPROVED** [review]
LGTM
